serverless/cmd/integrate/main.go

```diff
@@ -18,48 +18,140 @@ package main
 
 import (
 	"flag"
+	"io/ioutil"
+	"net/url"
+	"os"
 
 	"github.com/golang/glog"
+	"github.com/google/trillian-examples/serverless/api"
 	"github.com/google/trillian-examples/serverless/internal/log"
 	"github.com/google/trillian-examples/serverless/internal/storage/fs"
 	"github.com/google/trillian/merkle/rfc6962/hasher"
+	"golang.org/x/mod/sumdb/note"
 
 	fmtlog "github.com/google/trillian-examples/formats/log"
 )
 
 var (
-	storageDir = flag.String("storage_dir", "", "Root directory to store log data.")
+	storageDir  = flag.String("storage_dir", "", "Root directory to store log data.")
+	initialise  = flag.Bool("initialise", false, "Set when creating a new log to initialise the structure.")
+	pubKeyFile  = flag.String("public_key", "", "Location of public key file.")
+	privKeyFile = flag.String("private_key", "", "Location of private key file.")
 )
 
 func main() {
 	flag.Parse()
 	h := hasher.DefaultHasher
 
+	// Read log public key from file or environment variable
+	var pubKey string
+	if len(*pubKeyFile) > 0 {
+		pubKeyURL, err := url.Parse(*pubKeyFile)
```

Comment:
I think you can just use `ioutil.ReadFile(*pubKeyFile)` directly and avoid using the URL stuff.

Reply:
done